utils/docker/README.md

```diff
@@ -21,22 +21,34 @@ to build, test and optionally measure test coverage of any pull request to memki
 For fully testing MEMKIND_DAX_KMEM, ndctl library version tag must be passed as parameter,
 see https://github.com/pmem/ndctl/tags.
 
+* **ENABLE_HWLOC** - install hwloc library.
+For fully testing MEMKIND_HBW*, MEMKIND_HIGHEST_BANDWIDTH, MEMKIND_LOWEST_LATENCY, MEMKIND_*LOCAL hwloc library version must be installed.
```

Comment:
Please change to: To fully test

"hwloc library version must be installed" - a specific version or any version of the library?

Reply:
Done in #507 ( after merge #507 it will be available in this PR)